Sources/Testing/EntryPoints/ABIv0/ABIv0.swift

```diff
@@ -0,0 +1,12 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+/// A namespace for ABI version 0 symbols.
+enum ABIv0: Sendable {}
```

Comment:
Not for right now, but if/when we expose this outside the module I'd like to bikeshed alternative naming conventions

Reply:
This enum is definitely not ever meant to be _API_. It's a namespace for ABI-version-0 symbols only.

I'm open to alternatives.